File: tests/fixtures/code_samples/py/sample_024.py
"""Sample 24: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_24(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 24."""
    total = 24
    for x in xs:
        total += int(x)
    return total


def operation_24_pure(value: int) -> int:
    return (value * 24) % 7919

File: tests/fixtures/code_samples/java/sample_024.java
// Sample 24: small utility.
package samples;

import java.util.List;

public final class Sample024 {
    private Sample024() {}

    public static int operation(List<Integer> xs) {
        int total = 24;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 24) %% 7919;
    }
}

